easybuild/easyconfigs/g/GDRCopy/GDRCopy-2.1-GCCcore-9.3.0-CUDA-11.0.2.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDRCopy'
+version = '2.1'
+local_cudaversion = '11.0.2'
+versionsuffix = '-CUDA-%s' % local_cudaversion
+
+homepage = 'https://github.com/NVIDIA/gdrcopy'
+description = "A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology."
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'NVIDIA'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['cecc7dcc071107f77396f5553c9109790b6d2298ae29eb2dbbdd52b2a213e4ea']
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('Autotools', '20180311'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('CUDAcore', local_cudaversion, '', True),
+    ('Check', '0.15.2'),
+]
+
+# Runtime system dependencies:
```

Comment:
@lexming I would clarify the comment a bit to mention why we don't include these as OS dependencies?

Bonus points for also mentioning the error message you get when trying to use this on a system that doesn't have the necessary kernel modules installed.

Reply:
@lexming I'd like to merge this but this comment hasn't been resolved yet, writing here just in case you missed it

Reply:
@bartoldeman thanks for the reminder, I totally missed it

@boegel I added the reason to not include the kernel modules as dependencies. Regarding the error, I checked and there is no error. The default behaviour of UCX in such a case is just to not use `gdr_copy` as transport.